CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.13.2
+
+* Fix a bug in `StreamQueue.startTransaction()` and related methods when
+  rejecting a transaction that isn't the oldest request in the queue.
+
```

Comment:
Ah yes, makes sense - you can reject a transaction before it gets its first event.